Apromore-Extras/OpenXES/src/test/java/org/deckfour/xes/CloneAndRemoveTest.java

```diff
@@ -1,4 +1,4 @@
-/*-
+/**
```

Comment:
This isn't javadoc for the package, so it shouldn't be a doc comment, just a normal one.

Reply:
done.